CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+Starting after Rigbox 2.2.0, this file contains a curated, chronologically ordered list of notable changes made to the master branch. This changlog is based on [keep a changelog](https://keepachangelog.com)
+
+## [Latest](https://github.com/cortex-lab/Rigbox/commits/master)
```

Comment:
May be good to add version in brackets after 'Latest' here so we don't forget whether it was a minor or major update

Reply:
I thought of 'Latest' as the most recent commits which aren't in a release yet

Reply:
The current version needs to be recorded somewhere.  Each time you push a fix it would be good to iterate the current version number, e.g. from `v2.0.0` to `v2.0.1`. This would be useful particularly for users not using Git to keep track of what version they're on.

Reply:
Also by the time you come to archive the current version you may not remember whether it was a major or minor release let alone how many patches were applied.

Reply:
how's it now? I also removed the subheaders - I think they're unnecessary and that the information they provide can just be mentioned in the bullet points